client/src/document/code/ai-explain.ts

```diff
@@ -0,0 +1,175 @@
+import { SSE } from "sse.js";
+
+function explain(context, callback) {
```

Comment:
types for the request to and the response from the server would be nice, but not a blocker